apps/homescreen/bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "homescreen",
```

Comment:
I don't think the name in bower.json is visible to the user, is it? Either way I've changed it.